recipes/belay/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "belay" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/belay-{{ version }}.tar.gz
+  sha256: 3a879c51157d4dc9a435ddd0ae19826b23e1b90409fbfd209b0861abbe29ff41
+
+build:
+  noarch: python
+  script:
+    - {{ PYTHON }} -c '__import__("os").unlink("pyproject.toml")'
```

Please remove this.

Why are you adding it?

---

This is added to remove pyproject.toml as per recommendation from @bollwyvl to build without poetry.

---

```suggestion
    - rm pyproject.toml  # Use setup.py until required poetry version is out of beta 
```
Do something less esoteric and/or write a comment explaining to readers why something is unusual is happening, so that reviewers have an easier job.

---

```suggestion
    - {{ PYTHON }} -c '__import__("os").unlink("pyproject.toml")'  # [unix]
    - del pyproject.toml  # [win]
```